jest.config.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  transform: {
+    ".(ts|tsx)$": "ts-jest"
+  },
+  testPathIgnorePatterns: ["/node_modules/", "/dist/", "/test/declarations.d.ts"],
+  testRegex: "(/test/.*(spec).*|\\.(test|spec))\\.(ts|tsx|js)$",
+  moduleNameMapper: {
+    "\\.(css)$": "identity-obj-proxy"
```

Comment:
Is this needed since we don't have any css files?

Reply:
Yes, if we remove it we get the following error:

```
 FAIL  test/contextValidation.spec.tsx
  ● Test suite failed to run
/Users/Cesar/apps/dorg/daostack/DAOcomponents/node_modules/loaders.css/loaders.css:14
    @-webkit-keyframes scale {
    ^

    SyntaxError: Invalid or unexpected token
```